queries/sql/highlights.scm

```diff
@@ -160,18 +163,54 @@
   (keyword_like)
   (keyword_similar)
   (keyword_over)
+  (keyword_change)
+  (keyword_modify)
+  (keyword_restrict)
+  (keyword_first)
+  (keyword_after)
+  (keyword_range)
+  (keyword_rows)
+  (keyword_groups)
+  (keyword_unbounded)
+  (keyword_preceding)
+  (keyword_following)
+  (keyword_exclude)
+  (keyword_current)
+  (keyword_ties)
+  (keyword_others)
+  (keyword_unique)
+  (keyword_preserve)
+  (keyword_zerofill)
+  (keyword_format)
+  (keyword_fields)
+  (keyword_row)
+  (keyword_sort)
+  (keyword_compute)
+  (keyword_stats)
+  (keyword_comment)
+  (keyword_partitioned)
+  (keyword_location)
+  (keyword_cached)
+  (keyword_lines)
+  (keyword_stored)
 ] @keyword
 
+(keyword_with) @repeat
+
+(keyword_as) @label
```

Comment:
What's the reasoning for using repeat & label here?

Reply:
This was an attempt to diversify keywords a bit.

So `with` was probably not a good idea, since it just occurs once at the start of CTEs and then only in different locations (e.g. in hive create tables...) So I would revert that to a normal `keyword`

`keyword_as` as a `@label` since it creates an alias and this can be seen as a `label`. However, this is probably not necessary and purely cosmetic.

I think I will revert both changes.

Reply:
labels are for literal identifiers that are marked as a label, such as `goto Foo` where Foo is a label

Reply:
I removed these two queries